test/services/embedding/enhanced-service.test.ts

```
@@ -0,0 +1,285 @@
+      // Access private method for testing
+      const prepareFunctionText = (service as any).prepareFunctionText.bind(service);
+      const text = prepareFunctionText(mockFunctions[0]);
+      
+      expect(text).toContain('Primary: A test function for unit testing');
+      expect(text).toContain('Function: testFunction');
+      expect(text).toContain('Signature: function testFunction(): void');
+    });
+
+    it('should handle JSDoc when no description exists', () => {
+      const funcWithoutDesc = { ...mockFunctions[1] };
+      delete funcWithoutDesc.description;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)